crac-plugin/src/main/resources/warmup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Warm up the app.  For now, just call curl, we may want a way for this to be configurable
+curl http://localhost:8080
```

Comment:
This would fail if the user configures a different port for the Micronaut application, right?

Reply:
Yes, thats why I allow the user to specify their own warmup script to replace this default...

I need to add the same for "readiness"

Reply:
Readiness handled here https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/524/commits/2a0cb2dfb6d0c0e4a90c441467289a843552766b